stdpopsim/cli.py

```diff
@@ -649,7 +745,7 @@ def run_simulation(args):
             contig=contig,
             samples=samples,
             dfe=args.dfe,
-            dfe_interval=dfe_interval,
+            dfe_interval=dfe_summary_str,  # not sure what we want to print
```

Comment:
me neither, here we would want to print I guess:

- if the string "1000,2000", then just: "[1000,2000)"
- if a bed file, then np.min(array), np.max(array)
- if interval is not defined then contig length

Reply:
think this is all set, however i'm not dealing with the bed file in this PR.... not sure how to stage those merges

Reply:
I think what you're printing is good?